server/channels/api4/job_test.go

```diff
@@ -110,21 +116,40 @@ func TestGetJobs(t *testing.T) {
 		defer th.App.Srv().Store().Job().Delete(job.Id)
 	}
 
-	received, _, err := th.SystemAdminClient.GetJobs(context.Background(), 0, 2)
-	require.NoError(t, err)
+	t.Run("Get 2 jobs", func(t *testing.T) {
+		received, _, err := th.SystemAdminClient.GetJobs(context.Background(), 0, 2, "", "")
+		require.NoError(t, err)
 
-	require.Len(t, received, 2, "received wrong number of jobs")
-	require.Equal(t, jobs[2].Id, received[0].Id, "should've received newest job first")
-	require.Equal(t, jobs[0].Id, received[1].Id, "should've received second newest job second")
+		require.Len(t, received, 2, "received wrong number of jobs")
+		require.Equal(t, jobs[3].Id, received[0].Id, "should've received newest job first")
+		require.Equal(t, jobs[2].Id, received[1].Id, "should've received second newest job second")
+	})
 
-	received, _, err = th.SystemAdminClient.GetJobs(context.Background(), 1, 2)
-	require.NoError(t, err)
+	t.Run("Get oldest job using paging", func(t *testing.T) {
+		received, _, err := th.SystemAdminClient.GetJobs(context.Background(), 1, 3, "", "")
+		require.NoError(t, err)
+		require.Equal(t, jobs[1].Id, received[0].Id, "should've received oldest job last")
+	})
```

Comment:
Yeah you're right. For some reason we are handling the offset logic by multiplying in the app layer instead of the store layer. Which is leading to some confusion here. See `GetJobsByTypesPage` in the app layer which expects page & perPage but then the similarly named store function `GetAllByTypesPage` expects offset and limit. It's confusing, I'll try tidy it up a bit tomorrow

Reply:
Other parts of the codebase have the offset logic in the store layer. Should we also move the offset logic for jobs into the store?

Reply:
There was a bug with my paging where I wasn't multiplying the page*perPage. I've matched how the other job functions are doing it for now but we can create a separate ticket to refactor the paging. I started doing it but there were a lot changes and I didn't want to introduce too much in this PR